src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import CatCatalogue from "./components/CatCatalogue";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <CatCatalogue></CatCatalogue>
```

Comment:
[nit/tip] If a tag does not contain any children, you can shorten it like this
```suggestion
      <CatCatalogue />
```
(this also is used for example for `<img />` or `<input />` etc)